flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/SqlClientOptions.java

```diff
@@ -54,11 +55,13 @@ private SqlClientOptions() {}
 
     // Display options
 
-    @Documentation.TableOption(execMode = Documentation.ExecMode.STREAMING)
+    @Deprecated
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
     public static final ConfigOption<Integer> DISPLAY_MAX_COLUMN_WIDTH =
```

Comment:
There are still several places (e.g., `ResultDescriptor#maxColumnWidth`) using this configuration. Could you update to use `TableConfigOptions.DISPLAY_MAX_COLUMN_WIDTH` instead?

Reply:
It seems the `ResultDescriptor#maxColumnWidth` is still using the deprecated config.

Reply:
It is really difficult to use cross modules ConfigOptions and keep them synchronized. There are options:

option 1: use SqlConfigOption#DISPLAY_MAX_COLUMN_WIDTH as the fallback key of TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH. This does not work because flink-table-api-java does not depends on flink-sql. Adding new dependency will trigger a cycle dependency issue.
option 2: use TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH as the fallback key of SQLConfigOption#DISPLAY_MAX_COLUMN_WIDTH. If ResultDescriptor uses TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH, any change of sql-client.display.max-column-width will not be able to work.